src/components/dashboard/components/dictionary/ListDictionaries.jsx

```diff
@@ -1,39 +1,46 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import Card from './DictionaryCard';
-import Loader from '../../../Loader';
+import React from "react";
```

Comment:
Imports should still be  in single quotes

Reply:
True